js/adapt-stateful-session.js

```diff
@@ -161,7 +165,25 @@ export default class StatefulSession extends Backbone.Controller {
     logging.info(`course._isComplete: ${courseComplete}, course._isAssessmentPassed: ${assessmentPassed}, ${this._trackingIdType} completion: ${completionString}`);
   }
 
+  initializeContentObjectives() {
+    Adapt.contentObjects.forEach(model => {
+      if (model.isTypeGroup('course')) return;
+      const id = model.get('_id');
+      const description = model.get('title') || model.get('displayTitle');
+      offlineStorage.set('objectiveDescription', id, description);
+      if (model.get('_isVisited')) return;
+      const completionStatus = 'not attempted';
```

Comment:
should 'not attempted' be added as an enum to COMPLETION_STATE?

Reply:
there's at least one "incomplete" that can also be derived from the enum